tests/phpunit/api/v4/Action/AutocompleteTest.php

```diff
@@ -54,6 +54,11 @@ public function on_civi_api_prepare(\Civi\API\Event\PrepareEvent $event) {
     }
   }
 
+  public function setUpHeadless() {
+    // TODO: search_kit should probably be part of the 'headless()' baseline.
+    return \Civi\Test::headless()->install(['org.civicrm.search_kit'])->apply();
```

Comment:
This should work for getting `5.57` to a passing state. For `master`, we might move this to the parent (`Api4TestBase::setUpHeadless()`) or revise the definition of `headless()`.